pcstat/process.go

```diff
@@ -0,0 +1,54 @@
+// ps provides an API for finding and listing processes in a platform-agnostic
+// way.
+//
+// NOTE: If you're reading these docs online via GoDocs or some other system,
+// you might only see the Unix docs. This project makes heavy use of
+// platform-specific implementations. We recommend reading the source if you
+// are interested.
```

Comment:
^^ did this code come from another package? Can you please provide a link?

(sorry for the delay in reviewing this!)

Reply:
Yeah, this code mostly come from go-ps. I added `RSS() int` function for Process interface.
https://github.com/mitchellh/go-ps/blob/master/process.go